app/uk/gov/hmrc/personaldetailsvalidation/endpoints/PersonalDetailsCollectionController.scala

```diff
@@ -27,11 +27,11 @@ class PersonalDetailsCollectionController @Inject()(page: PersonalDetailsPage,
                                                     personalDetailsSubmission: FuturedPersonalDetailsSubmission)
   extends FrontendController {
 
-  def showPage(implicit completionUrl: CompletionUrl): Action[AnyContent] = Action { implicit request =>
-    Ok(page.render)
+  def showPage(implicit completionUrl: CompletionUrl, alternativeVersion: Boolean = false): Action[AnyContent] = Action { implicit request =>
```

Comment:
Do you need a default value here? I guess the app.routes file has a default value already.